src/resources/Dissolution/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from '@/resources/Dissolution/BC'
+export * from '@/resources/Dissolution/BEN'
+export * from '@/resources/Dissolution/CC'
+export * from '@/resources/Dissolution/CP'
+export * from '@/resources/Dissolution/GP'
+export * from '@/resources/Dissolution/SP'
+export * from '@/resources/Dissolution/ULC'
```

Comment:
This file and the others below are actually only slightly changed from before, but I renamed the folder they're in.